### PR TITLE
Improve flow of install page

### PR DESCRIPTION
**Pull request by dstansby**
# Description
- If I'm installing the bundled app, I don't care about pre-requisites for installing the Python package, so move that section under the "Installing as a Pytho package" section
- Modify the header levels, so the second level now only has "Install as Python package", "Install as bundled app", "Next steps". In particular, two sections which are only relevant to the Python package route have been moved down a level to simplify the top level contents.

### CODE REVIEW

**Comment by melissawm** [comment]
It looks like the pr-milestone-check is causing the CircleCI redirector action to be skipped. In any case, this looks good to me, thanks @dstansby !

**Review by melissawm — APPROVED** [review]

File: docs/tutorials/fundamentals/installation.md [hunk 1]
```diff
@@ -42,15 +39,6 @@ You may also want:
...
-### Prerequisites for installing napari as a bundled app
-This is the easiest way to install napari if you only wish to use it as a standalone GUI app.
-This installation method does not have any prerequisites.
-
-[Click here](#install-as-a-bundled-app) to see instructions
-for installing the bundled app.
-
-## Install as Python package (recommended)
-
 Python package distributions of napari can be installed via `pip`, `conda-forge`, or from source.
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/installation.md:42]:
```suggestion
````{note} 
While we recommend installing as a Python package, you can also install napari as [a conda bundled app](#install-as-a-bundled-app).
````

Python package distributions of napari can be installed via `pip`, `conda-forge`, or from source.
```


I agree with the logic, but I'd like to keep the early mention of the bundle so it's a bit more accessible. How's this for a compromise?

File: docs/tutorials/fundamentals/installation.md [hunk 2]
```diff
@@ -42,15 +39,6 @@ You may also want:
 - an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
 [venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
```
Comment (psobolewskiPhD) [inline docs/tutorials/fundamentals/installation.md:41]:
```suggestion

````{note} 
While we recommend installing as a Python package, you can also install napari as [a conda bundled app](#install-as-a-bundled-app).
````


I agree with the logic change, but maybe we could still mention the bundled app here? 
How's this for a compromise?

Edit: I hope the suggestion is behaving...

**Comment by psobolewskiPhD** [comment]
The whole CircleCI job isn't proc'ing...

**Review by psobolewskiPhD — APPROVED** [review]
I can't make the suggestion, but what are y'alls thoughts on using tab sets instead of the admonitions? (new line 58)

Like here:
https://napari.org/dev/developers/contributing.html#setting-up-a-development-installation
Code:
https://github.com/napari/docs/blob/1a3192fd7fc549b4ef8730dccae8bcbc673096c9/docs/developers/contributing.md?plain=1#L31C5-L54

**Comment by melissawm** [comment]
Love the tabs, maybe in a follow-up?

**Pull request merged**